configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,190 @@
+* Take note of the IP configuration of the ethernet network the Raspberry Pi is connected to: 
+    * In this document, we assume IP network `10.10.0.0/24` is configured on the ethernet LAN, and the Raspberry Pi is to manage IP network `192.168.4.0/24` for wireless clients.
+    * Please select another IP network for wireless, e.g. `192.168.10.0/24`, in case IP network `192.168.4.0/24` is already in use by your ethernet LAN.
+* Have a wireless client (laptop, smartphone, ...) ready to test your new access point.
+
+<a name="software-install"></a>
+## Install the access point and network management software
+
+In order to work as an access point, the Raspberry Pi needs to have the `hostapd` access point software package installed:
+
+```
+sudo apt install hostapd
+```
+Enable the wireless access point service to start when your Raspberry Pi boots:
+
+```
+sudo systemctl unmask hostapd
+sudo systemctl enable hostapd
+```
+
+In order to provide network management services (DNS, DHCP) to wireless clients, the Raspberry Pi needs to have the `dnsmasq` software package installed:
+
+```
+sudo apt install dnsmasq
+```
+Finally, install `netfilter-persistent` and its plugin `iptables-persistent`. This utilty helps saving firewall rules and restores them when the Raspberry Pi boots:
+
+```
+sudo DEBIAN_FRONTEND=noninteractive apt install -y netfilter-persistent iptables-persistent
+```
+Software installation is complete. We will configure the software packages later on.
+
+<a name="routing"></a>
+## Setup the network router
+
+The Raspberry Pi will run and manage a stand-alone wireless network. The Raspberry Pi will also route between wireless and ethernet networks, providing Internet access to wireless clients. At your option, you can skip routing and run the wireless network in complete isolation. 
+
+### Define the wireless interface IP configuration
+
+In this document, we assume IP network `10.10.0.0/24` is configured on the ethernet LAN, and the Raspberry Pi will manage IP network `192.168.4.0/24` for wireless clients.
+*Note:* Please select another IP network for wireless, e.g. `192.168.10.0/24`, in case IP network `192.168.4.0/24` is already in use by your ethernet LAN.
+
+The Raspberry Pi runs a DHCP server for the wireless network, this requires static IP configuration for the wireless interface (`wlan0`) in the Raspberry Pi. 
+The Raspberry Pi acts as router on the wireless network, as customary we will give it the first IP address in the network: `192.168.4.1`.
+
+To configure the static IP address, edit the configuration file for `dhcpcd` with:
+
+```
+sudo nano /etc/dhcpcd.conf
+```
+
+Go to the end of the file and add the following:
+
+```
+interface wlan0
+    static ip_address=192.168.4.1/24
+    nohook wpa_supplicant
+```
+
+### Enable routing and IP masquerading
+
+This section configures the Raspberry Pi to let wireless clients access computers on the main network, and from there the Internet.
```

Comment:
To avoid any ambiguity, maybe it's worth specifically referring to the "ethernet network" (or "wired network"), rather than the "main network"?

Reply:
Nitpicking a bit, nothing says that there isn't a wireless bridge in the main network.
Also see above https://github.com/raspberrypi/documentation/pull/1268#discussion_r346414657 my proposal for using "main/secondary networks".
But yes, if the sentence is unclear, by all means amend it.